.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md

```diff
@@ -0,0 +1,64 @@
+---
+name: "Feature Request"
+about: "Give us suggestions of features we could add to Terra."
+title: "[Feature] <Put your title here>"
+labels: "Type: Enhancement, Status: Pending"
+assignees: ""
+
+---
+
+<!--
+########################################################################################
+## WARNING!                                                                           ##
+## IGNORING THE FOLLOWING TEMPLATE WILL RESULT IN YOUR FEATURE REQUEST BEING CLOSED   ##
+########################################################################################
+-->
+
+### Pre-Request Checklist
+
+<!--
+  Please go through this checklist item by item and make sure you have successfully completed each of these steps.
+    - You must be on the LATEST version of Terra to make sure your feature hasn't been added yet.
+    - Make sure that there are no already existing feature requests similar to yours. (Including closed!) If you open a duplicate, it will be closed as such.
+    - Make sure that this is actually in the scope of Terra.
+    - Make sure that this is not a feature request that should be made for a specific Terra *pack*, and instead applies to all of Terra.
+    - Make sure that you attach a copy of the latest.log file, if there are any exceptions thrown in the console.
+      Putting *just* the exception IS NOT ENOUGH. We need to be able to check that there wasn't anything else before that caused it.
```

Comment:
Add something about *specific* compatibilities (e.g. "Compatibility with TechCraft v7") will be ignored, and should be implemented in addons. *General* compatibilities (e.g. "Ability for features to be pulled from parent biomes") will be considered in the main project.